charts/skypilot/templates/api-deployment.yaml

```diff
@@ -313,24 +342,32 @@ spec:
           echo "Setting up AWS credentials..."
           if [ -n "$AWS_ACCESS_KEY_ID" ] && [ -n "$AWS_SECRET_ACCESS_KEY" ]; then
             echo "AWS credentials found in environment variables."
-            aws configure set aws_access_key_id "$AWS_ACCESS_KEY_ID"
-            aws configure set aws_secret_access_key "$AWS_SECRET_ACCESS_KEY"
+            mkdir -p /root/.aws
+            # Create credentials file manually (don't rely on aws CLI being installed)
+            cat > /root/.aws/credentials << EOF
+          [default]
+          aws_access_key_id = $AWS_ACCESS_KEY_ID
+          aws_secret_access_key = $AWS_SECRET_ACCESS_KEY
+          EOF
+            chmod 600 /root/.aws/credentials
```

Comment:
Prefer to just leave this as is. Is the AWS CLI not installed in the docker image??